python_modules/dagster/dagster/_core/storage/cloud_storage_compute_log_manager.py

```diff
@@ -21,16 +22,48 @@
     IO_TYPE_EXTENSION,
     LocalComputeLogManager,
 )
+from dagster._utils import ensure_file
 from dagster._utils.error import serializable_error_info_from_exc_info
 
 SUBSCRIPTION_POLLING_INTERVAL = 5
 
 
+@contextmanager
+def _truncate_file(path, max_bytes: int) -> Iterator[str]:
+    dest = tempfile.NamedTemporaryFile(mode="w+b", delete=False)
```

Comment:
there are some funky things with NamedTemporaryFile on windows to watch out for here https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile

seems like setting delete=False resolves them though? But do we need to worry at all about this file sticking around and using up disk e.g. when running dagster locally?